src/file.cr

```diff
@@ -360,9 +358,7 @@ class File < IO::FileDescriptor
     end
 
     String.build do |str|
-      {% if !flag?(:windows) %}
-        str << SEPARATOR_STRING
-      {% end %}
+      str << SEPARATOR_STRING
       items.join SEPARATOR_STRING, str
     end
```

Comment:
Why `String.build` vs `SEPARATOR_STRING + items.join SEPARATOR_STRING`?

Reply:
`SEPARATOR_STRING + items.join SEPARATOR_STRING` creates two strings (one to join, the other for the `+`). `String.build` will just create one string.